.travis.yml

```diff
@@ -94,6 +94,7 @@ jobs:
         - $HOME/Library/Caches/pip/
 
   - os: windows
+    if: branch = master
```

Comment:
Pyccel no longer uses travis for its CI tests. To activate/deactivate certain tests you need to use this file:
https://github.com/pyccel/pyccel/blob/master/.github/workflows/Github_pytest.yml

Reply:
That said, unless you only have a problem on a specific platform it is better to test locally as you can pyccelize the file yourself and see what the error is. The pyccel tests give very little information when the compilation fails

Reply:
You can get rid of all changes to this file using `git checkout master -- .travis.yml`